sunspot/lib/sunspot/search/pivot_facet.rb

```diff
@@ -0,0 +1,67 @@
+module Sunspot
+  module Search
+    class PivotFacet
+      class Row
+        def initialize(fields, pivot, setup)
```

Comment:
I believe fields  should be in last because you are expecting more than 1 values.               
 def initialize(pivot, setup, fields).

Reply:
Hey @anikett-rpx, thanks for the comment!

Could you give me an example of what you mean? `fields` should be an explicit array of field names, entered like `.new([:field1, :field2, ...], ...)`.